app/scripts/modules/ecs/src/ecs.help.ts

```diff
@@ -83,6 +83,8 @@ const helpContents: { [key: string]: string } = {
     '<p>Designates how many tasks, at a minimum, to run on the specified capacity provider. Only one capacity provider in a capacity provider strategy can have a <em>base</em> defined.</p>',
   'ecs.capacityProviderWeight':
     '<p>Designates the relative percentage of the total number of tasks launched that should use the specified capacity provider.</p>',
+  'ecs.evaluateExpression':
+    '<p>Whether to evaluate <a href="https://spinnaker.io/guides/user/pipeline/expressions/"><b>pipeline expressions</b></a> within the task definition artifact in this stage. Checking this box let\'s you evaluate your task definition artifact coming from external sources.(e.g. GitHub) </p>',
```

Comment:
please add `target="_blank"` within the leading `<a>` tag to ensure the link opens in a new tab (see line 78 for example)